.github/workflows/deploy-docs.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
                 path: .cache
             - run: pip install -e ".[dev]"
             - run: ./scripts/build-docs.sh
-            - run: echo "VERSION=$(python3 -c 'from importlib.metadata import version; print(version("""faststream"""))')" >> $GITHUB_ENV
+            - run: echo "VERSION=$(python3 -c 'from importlib.metadata import version; print(".".join(version("faststream").split(".")[:2]))')" >> $GITHUB_ENV
```

Comment:
This will not work. VERSION itself is inside double quotes and then python lines are inside single quotes.
You need to use triple quotes for strings.